### DISCUSSION TITLE
[FEEDBACK REQUESTED] Which platform should we use for organising events?

### DISCUSSION THREAD

**Discussion by aleesteele** [the-turing-way/the-turing-way#4377] [Ideas]
### Summary

In January 2024, we switched our event management platform from Eventbrite to the built-in Zoom platform.

However we have really had trouble with the Zoom registration forms after testing them over the past ~6 months. 

If you haven't been following this ongoing conversation so far, we have some forms of documentation already:
- We've documented different platform options previously: https://docs.google.com/document/d/1tt2puVoBoVDnTtcPLxN_qoo2KhrtWFMRQR0fMWjU3I4/edit
- Here are a few issues about how we use open infrastructure: https://github.com/the-turing-way/the-turing-way/issues/3218#issuecomment-2032360282 & #2811 

Because we use the registration form for a variety of different types of events and in different contexts, from seminars-style discussions with the Fireside Chat, to our Book Dash, to onboarding calls and more... 

It's important that we use a platform that is open and accessible, but also contains the tools we need to track registrations, attendance analytics, etc.

## Feedback request

For ease of organisation (and out of urgency) for a few upcoming events, we will be using a combination of Eventbrite (for a fireside chat) and Google forms (for registrations for onboarding). 

But before we move forward in testing another platform for our registrations, we would love to keep track of your feedback and current experiences with platforms, in particular the following:

🌷 Eventbrite
🌻 Zoom Registrations
🐬 Humanitix
🌴 Google form
🍰 Indico

Would you be able to leave a comment here with you experience? 

Feel free to any or all of the following questions:
- Have you used any of these platforms: what worked? what didn't? what did you enjoy? what did you not?
- What platforms do you currently use and have experience with?
- If an organiser: what platform do you currently use to gather registrations? 
- If a participant: what do you prefer to receive when signing up for an event? (Reminder email, calendar invite, etc.)?

**Comment by aleesteele** [comment]
### Summary

In January 2024, we switched our event management platform from Eventbrite to the built-in Zoom platform. This has changed the workflow for setting up events, which I'll be documenting here as a checklist at the advice of @malvikasharan (thank you! 🙏)

Adding in some problems we've had here with the built-in Zoom platform over the past few months of using it for registrations after moving away from Eventbrite:
- **Collaborative editing**: Only the event host/owner is able to edit details. The event co-hosts are not able to see registrations, edit event information, and/or change anything about the event itself. 
- **Zoom registration loop:** At the February 2024 Community Call/Forum, participants were asked to sign-up, and then received a link that sent them back to the registration form. This is a [documented problem according to the Zoom community forums](https://community.zoom.com/t5/Meetings/Registered-participants-are-being-redirected-to-register-again/m-p/117346). 
- **Zoom registration error**: The Zoom form remained open for registrations after the event had started, instead of automatically changing into a splash page where people could join the meeting page automatically. The "close registration after meeting date" option was selected. 
- **Renaming participants**: During the Onboarding call in March 2024, the Zoom room automatically renamed all participants in the call to one person. The link from the registration form 
- **Follow-up emails to participants**: While there is an option to resend the Confirmation of Registrations, there is no able to send custom emails to participants as confirmation. In the confirmation of registration email, there are limited options in what can be sent. Custom message texts can be lost in the GUI of the automatic zoom message.
- **Customising form for registration**: There are limited options of what can be added to a Zoom registration form (i.e. no multiple choice answers). Adding custom messages. 
- **Keeping track of registrations**: There are a few issues with this that we had used with Eventbrite. The sign-up list cannot be easily exported for others to view or edit. Co-hosts cannot see the list of people who have signed up to be able to review registrations or answers (especially important if person who has created the zoom link, who may be hosting) needs support during the call.
- **Archiving event information more broadly**: As of 2 April, the Zoom archive of the zoom link and other registration information has disappeared for Community Call (#3510), but remains for Onboarding call (#3586). 
- **No calendar invite**: 

My suggestion here would be to switch to another platform for registrations – so we can avoid all of these inner-platform issues on Zoom, but will check back with the delivery team!

**Comment by aleesteele** [comment]
Notes from discussion with @vhellon (thank you for your help! 🙏 ):
- Possible to add a custom banner to email confirmation to keep branding similar
- Make sure everything is sent through zoom platform (may be responsible for some of the zoom)
- Search by meeting ID number to get archive
- Registration reports, attendees, attendance time, countries

**Comment by aleesteele** [comment]
Note from 1:1 check-in with @AoifeHughes: need to change documentation in the Community Handbook from Eventbrite to Zoom

**Comment by AoifeHughes** [comment]
I'll have a look at this later in the week, information is in the TTW hackmd

**Comment by aleesteele** [comment]
_Re-opening this issue after discussion about June 2024 Book Dash - as used for the Community Share-outs & Fireside Chat. This issue is more relevant for once we choose a platform, but has been added here to archive for now._ 

## Documenting Process for Community Forum and Onboarding Calls
_Adding this checklist - which will have to shift depending on the platform we choose_

### Checklist for using Zoom platform

### Before event
- [ ] Create event at set time
- [ ] Add co-hosts to call (Note: co-hosts cannot edit settings and/or register if they are made co-hosts)
- [ ] (If applicable): Double-check with co-hosts that they have accepted the invitation to co-host, as they cannot register for the event
- [ ] (If applicable): Add in description for event (Note: for data protection, we need to add the statement: "Data collected through this form is subject to the Alan Turing Institute's privacy policy. Learn more here: https://www.turing.ac.uk/privacy-policy".
- [ ] (If applicable): Check 'Registration form' option
- [ ] (If applicable): Check 'Close registration after meeting date' if applicable
- [ ] (If applicable): Upload banner (check if banner is right size and resolution)
- [ ] (If applicable): Manually add additional questions to sign-up form (we usually add: "What are you most interested in learning about during this call?"
- [ ] (If applicable): Manually move sign-up information to 3rd party storage platform for archiving (no option to export to csv file)
- [ ] (If applicable): Customise email for registrations (Note: the option here is to add a line of text – not a custom message). 
- [ ] (If applicable): Manually send automatically-generated reminder email from Zoom platform
- [ ] (If applicable): Use manually-exported list of registrations to send a reminder/invitation email to registrants before the event (if wanting to use a custom email - not template from Zoom)

### During event
- [ ] Make people/trainers/etc co-hosts of call (will have to manually do so if co-hosts have not accepted invitation to host call, and/or registered as attendees and not as hosts)
- [ ] Turn on transcriptions (used for subtitles)
- [ ] (If applicable) Ask for consent and share information about recording 

### After event
**Note: sometimes video and/or zoom events are deleted, so it's important to archive quickly.**
- [ ] Download video and subtitles
- [ ] Upload to Youtube or other
   - [ ] Add chapter markers
   - [ ] Add designed thumbnail
   - [ ] Add in event information to description
   - [ ] Correct transcriptions/subtitles

### What needs to be done?
- Add in the full workflow for using Zoom

**Comment by da5nsy** [comment]
I've used Indico recently (for Digital Research Academy events) and I've been really happy with it. 

The UI isn't the most intuitive or slick, but it works and I haven't found any bugs or glitches in it, and it has very broad functionality (I haven't found anything that I wanted to do that wasn't available, and I've found many extra bells and whistles that I hadn't even considered).

CCing @HeidiSeibold as she set it up and hosts it and so might have additional thoughts.

**Comment by bbartholdy** [comment]
I've used Eventbrite quite a lot and it generally works fine. They offer code (javascript) to embed the event registration in websites. The main limitation is that a free account only allows you to create events for up to 25 people.

**Comment by RichardJActon** [comment]
In the spirit moving to more open tools when the opportunity presents I would suggest considering [pretix](https://pretix.eu/about/en/) as I think they will have the featureset required. I would say that their licensing approach falls under the slightly controversial [fair-code](https://faircode.io/) category where it is mostly open but violates freedom zero / the 'no discrimination against fields of endeavor' clauses of the free software / open source standard definitions in that that is prohibits uses which directly commercially compete with their Saas offering. You can self-host it but not host an instance which you offer commercial use of to third parties, otherwise it is largely AGPL and Apache 2.0 terms.

Some fully open options:
- [mobilizon](https://mobilizon.org/en/) published by framasoft known to this community already through our use of framapad.
- [gathio](https://gath.io/) which uses the activitypub protocol to federate instances (the same underlying technology as mastodon for those unfamiliar) but they are relatively new and not very feature rich as of yet.

**Comment by HeidiSeibold** [comment]
I really enjoy working with indico and I also like it better than pretix, which is ok, but I don't like the wording ("tickets") in this context as much.

Indico is developed by CERN and used by organisations such as Unesco, so I think it's fair to say that it will stick around. 

I'd be happy to discuss sharing the Digital Research Academy instance, if that is an option for you https://events.digital-research.academy/
💜

**Comment by penyuan** [comment]
I'm personally very curious about mobilizon mentioned by @RichardJActon and Indico by @HeidiSeibold. On top of those, I'm aware of two more options: 

1. [Eventyay](https://eventyay.com) - Fully open source ([repo link](https://github.com/fossasia/open-event-server)), self-hostable and with official instance that you can just pick up and use. I know groups who successfully used Eventyay for organising big hybrid conferences such as the *huge* annual [FOSSASIA](https://eventyay.com/e/55d2a466). In fact I think some FOSSASIA organisers are also the Eventyay developers. 
2. [Ticketpass](https://ticketpass.org) - Claims to be an Eventbrite competitor that "donate 50% of our booking fee to charity and social causes". Out of principle, I cannot officially endorse this option because it is not open source, but listing it here for completeness!

**Comment by Lextuga007** [comment]
I've tried out Eventyay for events for https://github.com/nhs-r-community based on the suggestions here but found it was a little buggy. I couldn't get the tickets to show at first and had to set up a new event in the end, but now it logs me out without warning - used on Chrome and Edge. I'd be really interested to see it in action though as it looks potentially really good as it offers payment through PayPal, Strike and invoices (which is the main way we do payments in the Public Sector).

**Comment by Lextuga007** [comment]
To update I saw [Pretix](https://pretix.eu/about/en/) suggested here and have tested this out and have been very impressed. It's free for free events, open source and can manage quite complicated situations like a coffee and coding running on different days in different weeks. It's also possible to set up "season tickets" which we need for NHS-R Community as we have restricted event numbers for workshops so can only offer those to colleagues from certain organisations (public sector as an example). I'm currently going through my organisation's Information Governance and Cyber security for approval to use this platform and can share the outcomes and agreements once that's sorted. We are going to use the web based platform rather than hosting ourselves.

**Comment by aleesteele** [comment]
Hi @Lextuga007, @penyuan, @HeidiSeibold, @RichardJActon, @bbartholdy, @da5nsy - thank you all again for your thoughtful input on this issue!

During this time of transition for _The Turing Way_, we are currently using Zoom to host our meetings, and Google forms for collecting sign-ups to community events. While neither of these platforms are open source, we have been using them as interim tools while we develop a longer-term solution and plan for the community this year. We hope to transition to tools like Indico, Pretix, Eventyay etc when we're more stable.

With the @the-turing-way/community-management-working-group - we'll be revisiting this issue and infrastructure for event organising over the next few months. @LelleDi kindly added it to the group's project board.

Thanks everyone! 🙏

**Reply by ashatitus** [reply]
Hi @aleesteele, @RichardJActon, @SaraVilla and @RichardJActon 

Just to jump in - in thinking about tools to host meetings in the long run, it might be good to stay with mainstream tools like Zoom and Google. 

The danger of being exclusively open source is that tools like Indico, Pretix and Eventyay might be too inaccessible for the kind of global contributors that you want to attract and retain after the excitement of book dash events :-)! Alternatively, have at least have one 'big tech' option along with open source so that academic researchers outside the software development community can find the Turing Way and stay engaged.

**Comment by SaraVilla** [comment]
@JimMadge shall this be a discussion? shall I convert it?

**Comment by JimMadge** [comment]
> [@JimMadge](https://github.com/JimMadge) shall this be a discussion? shall I convert it?

Yes, feels like a discussion. It might be stale or solved though :eyes:.

**Comment by RichardJActon** [comment]
feels pretty discussion-y to me - it's a question that might come up again if there are any issues with current tools.

**Comment by SaraVilla** [comment]
Moving it to discussion and we can revisit whether closing it or action something else in some time
